src/components/Popup/BackupPage.vue

```diff
@@ -17,23 +17,47 @@
       <a-button-link
         download="authenticator.txt"
         :href="exportOneLineOtpAuthFile"
-        v-if="!unsupportedAccounts"
+        v-if="!unsupportedAccounts & isDataLinkSupported"
```

Comment:
Can we use blobl URLs (https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL) so that we can also enable backup feature on Safari?

Reply:
Unfortunately we don't have blob url support either on Safari.

Reply:
The only available on Safari for now is file download. I didn't figure out an approach for working around for other backup solutions yet.

Reply:
https://developer.apple.com/forums/thread/657490 seems a bug for Safari